site/team/laufen/trainer_laufen/trainer2/index.html

```diff
@@ -0,0 +1,22 @@
+<div class="container distance">
+  
+    </br>
+        <h2>Hey, ich bin die Karin Wagner</h2>
+        
+        <img src="/assets/images/team_pics/Laufen/2019KarinWagner_lowspec.jpg">
+        
+        
+        <p><strong>Funktion: </strong>Betreuerin von Lauf- und Walkinggruppen</p>
+        <p><strong>Name: </strong>Karin Wagner</p>
+        <p><strong>Mein Alter</strong>: AK60</p>
+        <p><strong>Meine liebste Sportart</strong>: -</p>
+        <p><strong>Meine liebste Trainingsform</strong></p>
+        <p><strong>Mein h&auml;rtester Wettkampf</strong>: -</p>
```

Comment:
can you decide wheter this shuld be a list or all other pages should not be a list so we dont have multiple layouts